src/systemctl/systemctl-show.c

```diff
@@ -705,11 +706,14 @@ static void print_status_info(
         if (i->memory_current != UINT64_MAX) {
                 printf("     Memory: %s", FORMAT_BYTES(i->memory_current));
 
+                bool show_memory_swap, show_memory_zswap_current;
```

Comment:
It's not necessary to split the initialization from declaration?

Reply:
I somehow as under the impression that this style was expected. Changed.